test/Microsoft.Dnx.TestHost.Tests/OSSkipConditionFacts.cs

```diff
@@ -47,5 +47,17 @@ public void RunTest_DoesNotRunOnWin7()
                 Assert.False(true, "Test should not be running on Win7");
             }
         }
+
+        [ConditionalFact]
+        [OSSkipCondition(OperatingSystems.AllWindows)]
+        public void TestSkipWindows()
+        {
+            var env = (IRuntimeEnvironment)CallContextServiceLocator
```

Comment:
Declare this in a class level variable and make all the test use it rather than duplicating it